app/controllers/orders_controller.rb

```diff
@@ -59,6 +59,20 @@ def order_cancel
     @payment.update(status: payment_status)
   end
 
+  def index
+    @orders = @current_user.orders.where.not(status: 'cart').includes(:line_items).order(id: :desc)
+  end
+
+  def cancel_order
+    @order.update_inventory(true)
```

Comment:
this should be done after canceling order

Reply:
Done